catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/CsexraAttribute.java

```diff
@@ -85,20 +129,159 @@
 
     static final CsexraAttribute SNOW_DEPTH_MIN =
             new CsexraAttribute(Isr.SNOW_DEPTH_MIN_CENTIMETERS,
-                    SNOW_DEPTH_CAT,
+                    SNOW_DEPTH_CAT_NAME,
                     getSnowDepthAccessorFunction(Pair::getLeft),
                     new IsrAttributes().getAttributeDescriptor(Isr.SNOW_DEPTH_MIN_CENTIMETERS),
                     "snowDepthMin",
                     ATTRIBUTE_NAME_PREFIX);
 
     static final CsexraAttribute SNOW_DEPTH_MAX =
             new CsexraAttribute(Isr.SNOW_DEPTH_MAX_CENTIMETERS,
-                    SNOW_DEPTH_CAT,
+                    SNOW_DEPTH_CAT_NAME,
                     getSnowDepthAccessorFunction(Pair::getRight),
                     new IsrAttributes().getAttributeDescriptor(Isr.SNOW_DEPTH_MAX_CENTIMETERS),
                     "snowDepthMax",
                     ATTRIBUTE_NAME_PREFIX);
 
+    static final CsexraAttribute SNOW_DEPTH_CAT = new CsexraAttribute("snow-depth-category",
+            SNOW_DEPTH_CAT_NAME,
+            tre -> TreUtility.convertToInteger(tre, SNOW_DEPTH_CAT_NAME),
```

Comment:
IDEA reports this as an illegal self-reference -- attempting to reference methods of a constructed object within the constructor.